apps/portals/adknowledgeportal/src/config/synapseConfigs/computational_tools.ts

```diff
@@ -18,18 +26,17 @@ export const computationalCardConfiguration: CardConfiguration = {
   labelLinkConfig: [
     {
       isMarkdown: false,
-      matchColumnName: 'grant',
-      URLColumnName: 'Grant Number',
+      matchColumnName: COMPUTATIONAL_TOOLS_COLUMN_NAMES.GRANT,
+      URLColumnName: PROGRAM_TABLE_COLUMN_NAMES.GRANT_NUMBER,
```

Comment:
I didn't manage to do this for all portals & all components but it would save a lot of confusion and risk for error if we added constants for the column names used throughout the config, especially when cross-linking between relations. I started doing that here.

Reply:
Good idea